install/README.md

```diff
@@ -1,3 +1,48 @@
 # Install
 
 Resources for installing Gitpod
+
+## Replicated/KOTS
+
+Gitpod has teamed up with Replicated to simplify the installation of Gitpod for self-hosted users. This provides an installation interface, simplified license delivery for both community and paying users, easier upgrades and improved
+
+Kubernetes Off-The-Shelf (KOTS) is the technology used to deliver and Replicated is the company behind it. The terms may be used interchangeably, but we will tend to talk of KOTS.
+
+### Channels
+
+> The `stable` channel should be used
+
+The `stable` channel represents tested and validated releases. This will typically receive one update per month with each [milestone](https://github.com/gitpod-io/gitpod/milestones). Hotfixes may be released during the month where necessary.
+
+The `beta` channel represents the release candidates for the `stable` channel. This will receive updates prior to a `stable` release ready for testing.
+
+The `unstable` channel receives every release that is pushed to the `main` branch, typically receiving multiple updates per day.
+
+#### Licenses
+
+A license is required to install via KOTS. For a full list of features, please see our [docs](https://www.gitpod.io/self-hosted). To obtain a license for a paid subscription, please go to the [enterprise license](https://www.gitpod.io/enterprise-license) section of our docs.
+
+Download the licenses below:
+
+ - [Stable](./licenses/Community.yaml) _&#8592; recommended_
+ - [Beta](./licenses/Community%20(Beta).yaml)
+ - [Unstable](./licenses/Community%20(Unstable).yaml)
+
+### Installation
+
+#### Existing Cluster
+
+```shell
+curl https://kots.io/install | bash
+kubectl kots install gitpod
```

Comment:
Guessing it would take `stable` by default. How about adding a reference on using the channel here i.e `gitpod/unstable`. Not sure how users would know otherwise 🤔

Reply:
Do you know what, even though the KOTS docs actually say that I tried putting `kubectl kots install gitpod/unstable` and using the `beta` license - and it seems to install whatever the license says, rather than whatever the install command says

Reply:
AH NICE! Makes sense! Then I think the installation command doesn't need any of this!

Reply:
Yeah, agreed. I've asked Barry and Todd from Replicated to cast their eyes over this to ensure I've not made any factual errors.